jobs/build-arch.Jenkinsfile

```diff
@@ -443,7 +443,7 @@ lock(resource: "build-${params.STREAM}-${basearch}") {
         parallelruns = [:]
         if (uploading) {
             // Kick off the Kola AWS job if we have an uploaded image and credentials for running those tests.
-            if (shwrapCapture("cosa meta --get-value aws") != "None" &&
+            if (shwrapCapture("cosa meta --get-value amis") != "None" &&
```

Comment:
Dynamic programming languages strikes again...no compilation time, but one pays later in debugging obvious typos etc. at runtime.